### ISSUE TITLE
[Eligibility Questionnaire][Accessibility] Progress bar links don't work

### ISSUE THREAD

**Issue by elenm** [CityOfDetroit/bloom#263]
The progress bar links in the application and eligibility questionnaire don't redirect to the appropriate page. 

If they're not meant to be links, we should change the mark up, so there is no confusion for screen reader and sighted users. If they should be links, we need to add functionality such that they redirect to the appropriate page. 

This involves updates to the Bloom component.
